test/common/http/codes_speed_test.cc

```diff
@@ -17,11 +17,14 @@
 namespace Envoy {
 namespace Http {
 
-class CodeUtilitySpeedTest {
+template <class SymTab> class CodeUtilitySpeedTest {
```

Comment:
nit: Even with the context of a symbol table PR, my first instinct was to read this as tab instead of table. Change this name?

Reply:
done --> SymbolTableClass.